tools/contractdeployer/main/main.go

```diff
@@ -19,4 +34,7 @@ func main() {
 	}
 	// print the contract address, to be read if necessary by the caller (important: this must be the last message output by the script)
 	fmt.Print(contractAddr)
+
+	// this is a safety sleep to make sure the output is printed from the docker container
+	time.Sleep(5 * time.Second)
```

Comment:
This sleep was moved from the `contract_deployer.go` code, I don't know if it's necessary but doesn't do much harm and if it's here then it doesn't pollute the core logic.